### PR TITLE
Make the license less restrictive. See #14

### PR DESCRIPTION
**Pull request by kvz**
So that people can use _just_ main.sh without bothering with also
distributing the license

### CODE REVIEW

File: main.sh [hunk 1]
```diff
@@ -24,8 +24,10 @@
 #
 #  LOG_LEVEL=7 ./main.sh -f /tmp/x -d
 #
-# Licensed under MIT
-# Copyright (c) 2013 Kevin van Zonneveld (http://kvz.io)
+# Licensed under MIT with a less restrictive clause that excepts you of
```
Comment (rouson) [inline main.sh]: Commenting on the grammar only, I suggest replacing "excepts you of" with "exempts you from".

File: main.sh [hunk 2]
```diff
@@ -24,8 +24,10 @@
...
+# redistributing the LICENSE file as long as you leave this comment in tact
```
Comment (bravo-kernel) [inline main.sh]: nitpicking: `intact`

**Comment by kvz** [comment]
Thanks @rouson. On second thought I think modifying the MIT License was foolish. Other folks who are not a lawyer will be suspicious of any deviations, even though my intentions are to make it even more permissive. As you say @rouson it matters a lot what the developer agrees to, I'm thinking of shipping the MIT license as-is, but stating in the code that you are not obligated to redistribute that file, as long as you leave the copyright comments with a reference to it intact. As for your personal question, the file that contains larger parts of b3pb should have these comments, and that's about it, afaic. Will work now on making the appropriate changes, including the typo fix that @bravo-kernel pointed out, thanks for that! 👍

**Comment by kvz** [comment]
A few iterations later, I think I'm happy with the results. Since I'm not changing the LICENSE, it has also become a less critical PR. I'll merge this already, but welcome additional comments/iterations here if you may have them.

**Pull request merged**